.env.local.example

```diff
@@ -0,0 +1,3 @@
+SENTRY_AUTH_TOKEN=
```

Comment:
Are we using this value anywhere?

I didn't see any usage of this `SENTRY_AUTH_TOKEN` value in this PR. 🤔

Reply:
Instead of including `SENTRY_AUTH_TOKEN` in the `sentry.properties` file, we can use environment variables for that. 

Reference: https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#use-a-properties-file